test/regress.c

```diff
@@ -223,6 +227,15 @@ test(char const * path, int limit, int resume)
       printf(" %u", (unsigned int) length);
     }
 
+    if (pkt_encryption == NESTEGG_PACKET_HAS_SIGNAL_BYTE_PARTITIONED) {
+      nestegg_packet_offsets(pkt, &pkt_partition_offsets, &pkt_num_offsets);
+      sha1_init(&s);
+      sha1_write(&s, (char const *) pkt_partition_offsets, pkt_num_offsets * 4);
```

Comment:
Use `sizeof(*pkg_partition_offsets)` rather than `4`

This is probably the wrong way to do it for big-endian compatibility though, because unlike the other places we're doing a `sha1_write` (at least, from a quick scan), this data has been converted to native endian during parsing, whereas the others are just hashing a byte buffer read directly from the file.

So maybe it's better to loop over the `pkt_partition_offsets` and printf them all individually, along with the `pkt_num_offsets`.